index.bs

```diff
@@ -798,97 +930,103 @@ Reflective modules are always already instantiated.
 1. Return _ns_.
 </emu-alg>
 
-<h2 id="local">Local Loading</h2>
+<h3 id="properties-of-the-module-constructor">Properties of the Reflect.Module Constructor</h3>
 
-<b>TODO:</b>
+The value of the \[[Prototype]] internal slot of the Reflect.Module constructor is the intrinsic object %FunctionPrototype%.
 
-<ul>
-<li>syntax for accessing module local information: <code>import local from this;</code>
-<li>dynamic import: <code>local.import()</code>
-<li>extending the hooks to handle <code>this</code>
-<li>debugging info
-<li>room for host environment-specific data
-</ul>
+Besides the internal slots and the length property (whose value is 0), the Reflect.Loader constructor has the following properties:
 
+<h4 id="Reflect.Module.evaluate">Reflect.Module.evaluate(m)</h4>
 
-<h2 id="browser">Browser Loader</h2>
+<b>TODO:</b> way to force evaluation of a module namespace exotic object (<code>Reflect.Module.evaluate(m)</code>? <code>m[Reflect.Module.evaluate]()</code>?)
+
+<h4 id="Reflect.Module.prototype">Reflect.Module.prototype</h4>
+
+The value of Reflect.Module.prototype is an ordinary object with a null [[Prototype]].
 
-<h3 id="browser-site-packages">Site Packages</h3>
+This property has the attributes { \[[Writable]]: false, \[[Enumerable]]: false, \[[Configurable]]: false }.
 
-The browser loader contains extra properties for storing <dfn>site packages</dfn>, an application-global set of globally available packages. These map in an internal table to unique URLs that in turn serve as keys in the module registry.
+<h3 id="module-prototype-object">Properties of the Reflect.Module Prototype Object</h3>
 
-<div class="note">
-<p>
-The site package system serves as a simple coordination mechanism for modest-sized applications, but it does not provide all functionality required of a full-fledged package management system. It is expected that development ecosystems will build around package management tools that deal with requirements outside the scope of this specification, such as version management and allowing multiple versions of a library to coexist with the same name.
-</p>
+<h4 id="Reflect.Module.prototype.constructor">Reflect.Module.prototype.constructor</h4>
```

Comment:
remove this -- there should not be a `constructor` property on the prototype (it can't have any properties).